barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/ultra_rollup_recursive_flavor.hpp

```diff
@@ -0,0 +1,144 @@
+#pragma once
```

Comment:
new recursive flavor - key difference is the IPA bool and indices in the VerificationKey